commands_config/group.go

```
@@ -0,0 +1,41 @@
+package commands_config
+
+import (
+	"github.com/assistant-ai/jess/commands_common"
+	"github.com/assistant-ai/jess/utils"
+	"github.com/assistant-ai/llmchat-client/client"
+	"github.com/urfave/cli/v2"
+	"os"
+)
+
+func DefineTestCommand(llmClient *client.Client, config *utils.AppConfig) *cli.Command {
+	testCommand := commands_common.JessCommand{
+		Command: &TestCommand{},
+	}
+	//configCommand := ConfigCommand{
+	//	Command: &ConfigCommand{},
+	//}
+
+	//configCommand := &ConfigCommand{}
+
+	modelNameMsg := "Model name:" + config.ModelName
+	logInfoMsg := "Log level:" + config.LogLevel
+	openaiKeyPath := "OpenAI API key path:" + config.OpenAiApiKeyPath
+	yourKey, _ := os.ReadFile(config.OpenAiApiKeyPath)
```

do not ignore errors